src/main/java/net/mcreator/element/types/Particle.java

```diff
@@ -55,7 +56,6 @@ public class Particle extends GeneratableElement {
 
 	public boolean canCollide;
 	public boolean alwaysShow;
-

```

Comment:
Please undo this unrelated change:

![image](https://github.com/MCreator/MCreator/assets/16374228/38ace88b-a1c3-4013-b00c-9fe6575c7c70)

Reply:
Ok

Reply:
Done!

Reply:
Fixed